### PR TITLE
Correct history on product page

### LINKED ISSUE #11281: Page isn't reloaded in FO when you change combination
**Describe the bug**
Page isn't reloaded in FO when you change combination

**To Reproduce**
1. Go to a product with combination in FO
2. Change combination
3. The URL change but the page isn't reloaded
4. If you click on previous buttons, you are back to the previous page instead of previous combination

**Additionnal information**
PrestaShop version: 1.7.5.0
PHP version: N/A

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Browser history wasn't working on product page when changing combination
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #11281 
| How to test?  | Go on product page and change combination, then try backward - frontward browser

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17133)

### CODE REVIEW

**Comment by PierreRambaud** [comment]
Could you squash my commit to remove `Test`  :sweat_smile: 
Since I worked on it, I can't approve :)

File: themes/_core/js/product.js [hunk 1]
```diff
@@ -32,6 +32,18 @@ var currentRequest = null;
 // Used to clearTimeout if user flood the product quantity input
 var currentRequestDelayedId = null;
 
+// Check for popState event
+var isOnPopStateEvent = false;
+
+<<<<<<< HEAD
```
Comment (PierreRambaud) [inline themes/_core/js/product.js]: Nope :scream:

**Comment by PierreRambaud** [comment]
Assets must be built?

**Review by PierreRambaud — DISMISSED** [review]
Need one more review, because I worked on it!

**Review by Progi1984 — DISMISSED** [review]

**Comment by SD1982** [comment]
@NeOMakinG could you please resolvve the themes/core.js conflict ?

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by SD1982** [comment]
the pull request correctly fixes the problem

**Before the pull request :**
https://drive.google.com/file/d/1Y30jZ8P9u2tsOt9PkSq52oxR-lfvLMJI/view?usp=sharing

**After the pull request :**
https://drive.google.com/file/d/18NJOS_fYx8AyV6bE8MyY6GRr_IXit_Tn/view?usp=sharing

**Comment by Progi1984** [comment]
Thanks @NeOMakinG

**Pull request merged**